scripts/steps/create-layout.js

```diff
@@ -241,7 +242,7 @@ function converter() {
         if (node.editUrl) {
           node.tagName = 'div'
           node.children = [
-            tag('h1', 'title', node.properties, node.children),
+            tag('h1', 'class_title', node.properties, node.children),
```

Comment:
Maybe I should rename css class for this `h1` if it confuses (here the word class refers to the API entity)

Reply:
Why do we need a new class?

Reply:
It is better to use `h2.subtitle` for content inside the group.

Or we can avoid using group title. We need it only for side menu.